.gitignore

```diff
@@ -32,3 +26,44 @@ server/src/entities/__pycache__/Settings.cpython-39.pyc
 server/src/prestart/__pycache__/setenv.cpython-39.pyc
 server/src/shared/__pycache__/LoggerFactory.cpython-39.pyc
 server/src/utils/__pycache__/generatePKCE.cpython-39.pyc
```

Comment:
@bfausett77 After adding "*.pyc", these specific .pyc files can be removed from this file.

Reply:
Done